Fix build failure with clang 8+:
error: invalid suffix on literal; C++11 requires a space between literal and identifier [-Wreserved-user-defined-literal]
--- src/wp/ap/gtk/ap_UnixApp.cpp.orig	2016-10-10 15:21:05.000000000 -0500
+++ src/wp/ap/gtk/ap_UnixApp.cpp	2019-10-09 09:42:29.000000000 -0500
@@ -863,7 +863,7 @@
 	if (len < (strlen(G_MODULE_SUFFIX) + 2)) // this is ".so" and at least one char for the filename
 		return false;
 	const char *suffix = file+(len-3);
-	if(0 == strcmp (suffix, "."G_MODULE_SUFFIX))
+	if(0 == strcmp (suffix, "." G_MODULE_SUFFIX))
 		return true;
 	return false;
 }
